frontend/src/pages/library/_itemId/index.vue

```diff
@@ -279,9 +293,7 @@ async function refreshItems(): Promise<void> {
   }
 }
 
-watch(() => (route.params as { viewId: string }).viewId, fetchLibrary, {
-  immediate: true
-});
+onMounted(() => fetchLibrary((route.params as { itemId: string }).itemId));
```

Comment:
```suggestion
onMounted(async () => fetchLibrary((route.params as { itemId: string }).itemId));
```

FYI, this is the only place where is good to stick on onMounted because the server is too slow serving the request. With a better item store (which will result in a similar behaviour to the one we currently have in the userLibraries and index), this will probably be solved, but we still would need the request to be done here, so data is refreshed, but items ofc will be loaded much faster. Exactly the same thing that happens in index.

Reply:
This only needs to be `async` if we need to `await` things, omitted for simplicity.